.githooks/post-checkout

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32169&commitId=36cbdac9f6d5e45673087951414d4e7d34c5c2a9&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|7| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|9| Changed|
|Avatar Converged.basic - RTL.normal.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>